flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/ProjectionTranslator.scala

```diff
@@ -136,4 +136,21 @@ object ProjectionTranslator {
       case e: PlannerExpression => e
     }
   }
+
+
+  /**
+    * Extracts the leading non-alias expression.
+    *
+    * @param expr              the expression to extract
```

Comment:
Please remove spaces. i.e.: "            the expression to extract" ->"the expression to extract"